.github/workflows/test-e2e.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           # Context for which we should look for the matching status
           statusName: ${{ (github.event_name == 'pull_request') && 'pull-lifecycle-mgr-build' || 'main-lifecycle-mgr-build' }}
-          timeoutSeconds: 600
```

Comment:
How was this value obtained? Trial and error?

Reply:
This is the timeout for waiting for image build, 10mins sometimes is not enough, you can find it in building history.